src/EnergyPlus/DaylightingManager.hh

```diff
@@ -100,7 +100,7 @@ namespace DaylightingManager {
         int const daylightCtrlNum, // zero if called for map points
         int const iRefPoint,
         int const loopwin,
-        DataDaylighting::CalledFor const CalledFrom, // indicate  which type of routine called this routine
+        Dayltg::CalledFor const CalledFrom, // indicate  which type of routine called this routine
```

Comment:
Can get rid of these entirely now (is it obvious that I do many of these changes using global search-and-replace?)